### PR TITLE
PR Added icons method to form

### PR DESCRIPTION
**Pull request by btepu002xamk**
Added icons method to form

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "@date-io/date-fns": "^2.14.0",
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
+    "@material-ui/icons": "^4.11.3",
```
Comment (villejuutila) [inline package.json:9]: Remove and use already existing MUI v5 icons package "@mui/icons-material"

File: src/components/editor/field-renderer/addable-field-renderer.tsx [hunk 2]
```diff
@@ -57,19 +55,7 @@ const AddableFieldRenderer: React.FC<Prop> = ({
...
-                border: "none"
-              },
-              "&.Mui-focused .MuiOutlinedInput-notchedOutline": {
-                border: "none"
-              },
-              "&:hover .MuiOutlinedInput-notchedOutline": {
-                border: "none"
-              }
-            }}
+            style={{ pointerEvents: "none" }}
```
Comment (villejuutila) [inline src/components/editor/field-renderer/addable-field-renderer.tsx:58]: Why is this file edited at all for this?

File: src/components/generic/form.tsx [hunk 3]
```diff
@@ -46,14 +47,14 @@ const Form: React.FC<Props> = ({
 
   /**
    * Method for rendering form icons
-   * TODO: Implement icons
+   * @param IconName Icon
```
Comment (villejuutila) [inline src/components/generic/form.tsx:50]: Add missing function param to TSDoc

**Review by villejuutila — CHANGES_REQUESTED** [review]
Maybe you should merge develop into this branch before merging this into develop.

File: src/components/generic/form.tsx [hunk 4]
```diff
@@ -14,6 +14,7 @@ import { useApiClient } from "app/hooks";
 import moment from "moment";
 import MetaformUtils from "utils/metaform-utils";
 import FormAutocomplete from "./form-autocomplete";
+import Icon from "@mui/material/Icon";
```
Comment (villejuutila) [inline src/components/generic/form.tsx:17]: Move to existing import from "@mui/material"

**Review by villejuutila — APPROVED** [review]

**Comment by WiljamiT** [comment]
LGTM

File: src/components/generic/form.tsx [hunk 5]
```diff
@@ -46,14 +46,15 @@ const Form: React.FC<Props> = ({
...
    */
-  const renderIcon = () => (
-    <Typography>
-      { strings.generic.notImplemented }
-    </Typography>
-  );
-
+  const renderIcon = (icon: IconName, key: string) => {
+    return (
+      <Icon key={key}>{icon}</Icon>
```
Comment (EVP1989) [inline src/components/generic/form.tsx:54]:
```suggestion
      <Icon key={ key }>{ icon }</Icon>
```

**Review by EVP1989 — APPROVED** [review]
Approved with suggestion

File: package-lock.json [hunk 6]
```diff
@@ -5,11 +5,13 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "essote-management-portal",
```
Comment (anttileppa) [inline package-lock.json:8]: Why package-lock has been updated? This PR does not have any changed to package.json so I would assume that also the lock-file should be untouched

**Review by anttileppa — CHANGES_REQUESTED** [review]

**Pull request merged**